tabpy-tools/client_tests/test_client.py

```diff
@@ -6,14 +6,15 @@
 except ImportError:
     from mock import Mock
 
-from tabpy_client.client import *
+from tabpy_tools.client import *
```

Comment:
Generally wildcard imports like this are frowned upon - any reason to not use explicit namespacing?